pybandits/base.py

```diff
@@ -187,3 +187,17 @@ def predict(self, forbidden_actions: Optional[Set[ActionId]] = None):
         probs: List[Dict[ActionId, float]] of shape (n_samples,)
             The probabilities of getting a positive reward for each action.
         """
+
+    def get_state(self) -> (str, dict):
+        """
+           Access the complete model internal state, enough to create an exact copy of the same model from it.
+           Returns
+           -------
+           model_class_name: str
+               The name of the class of the model.
+           model_state: dict
+               The internal state of the model (actions, scores, etc.).
+           """
+        model_name = self.__class__.__name__
+        state: dict = self.dict()
```

Comment:
Strategy is empty if you use self.dict() (look at the tests). Do you see it as a problem?

Reply:
No, it's not a problem, as this represents the state, and some strategies are stateless.